PocketKit/Sources/Sync/CoreDataInitializers/CollectionStory+CoreDataClass.swift

```diff
@@ -0,0 +1,32 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+import Foundation
+import CoreData
+
+@objc(CollectionStory)
+public class CollectionStory: NSManagedObject {
+    @available(*, unavailable)
+    public init() {
+        fatalError()
+    }
+
+    @available(*, unavailable)
+    public init(context: NSManagedObjectContext) {
+        fatalError()
+    }
+
+    internal override init(entity: NSEntityDescription, insertInto context: NSManagedObjectContext?) {
+        super.init(entity: entity, insertInto: context)
+    }
+
+    public init(
+        context: NSManagedObjectContext,
+        url: String
+    ) {
+        let entity = NSEntityDescription.entity(forEntityName: "CollectionStory", in: context)!
+        super.init(entity: entity, insertInto: context)
+        self.url = url
```

Comment:
Need to confirm with backend / android on using this as a key. Unclear on why we can't use `url` instead of `item.givenUrl` Open to any suggestions though!

https://mozilla.slack.com/archives/C05HR3FGFA9/p1689953416240889

Reply:
discuss via zoom, we will be using `url`